builtin/logical/database/backend.go

```diff
@@ -57,6 +60,20 @@ func Factory(ctx context.Context, conf *logical.BackendConfig) (logical.Backend,
 	b.credRotationQueue = queue.New()
 	// Load queue and kickoff new periodic ticker
 	go b.initQueue(b.queueCtx, conf, conf.System.ReplicationState())
+
+	// collect metrics on number of plugin instances
+	pluginInstanceGaugeProcess, err := metricsutil.NewGaugeCollectionProcess(
+		[]string{"secrets", "database", "backend", "pluginInstances", "count"},
+		[]metricsutil.Label{},
+		b.collectPluginInstanceGaugeValues,
+		metrics.Default(),
+		configutil.UsageGaugeDefaultPeriod, // TODO: we seem to have no way to access the actual configured values for these?
```

Comment:
There isn't a way to directly access Vault Core level config values from mounts. We'd probably have to thread it through `conf *logical.BackendConfig`, but I don't think common that we propagate global config values to mounts.

Reply:
It's probably fine then. We could always add a backend-specific config for it in the future.

Reply:
Should we delete or update the TODO note in the meantime?